what-is.asciidoc

```diff
@@ -12,24 +12,24 @@ The Ethereum platform enables developers to build powerful decentralized applica
 [[bitcoin_comparison]]
 === Compared to Bitcoin
 
-Many people will come to Ethereum with some prior experience of cryptocurrencies, specifically bitcoin. Ethereum shares many common elements with other open blockchains: a peer-to-peer network connecting participants, a consensus algorithm for synchronization of state (Proof-of-Work), a digital currency (ether) and a global ledger (the blockchain).
+Many people will come to Ethereum with some prior experience of cryptocurrencies, specifically bitcoin. Ethereum shares many common elements with other open blockchains: a peer-to-peer network connecting participants, a consensus algorithm for synchronization of state (proof-of-work system), a digital currency (ether) and a global ledger (the blockchain).
```

Comment:
We are keeping "Proof-of-Work" capitalized like that. Otherwise, we would have to change it everywhere. I'll leave that to the O'Reilly editors. Please revert this one change from the PR.